lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/error/ErrorResponse.java

```diff
@@ -31,7 +31,17 @@ public enum ErrorResponse {
     DCS_ERROR_HTTP_40X(1023, "DCS Responded with a HTTP Client Error status code"),
     DCS_ERROR_HTTP_50X(1024, "DCS Responded with a HTTP Server Error status code"),
     DCS_ERROR_HTTP_X(1025, "DCS Responded with an unhandled HTTP status code"),
-    TOO_MANY_RETRY_ATTEMPTS(1026, "Too many retry attempts made");
+    TOO_MANY_RETRY_ATTEMPTS(1026, "Too many retry attempts made"),
+
+    /**************************************DVA Specific Errors************************************/
+    FAILED_TO_PREPARE_DVA_PAYLOAD(1003, "Failed to prepare DCS payload"),
+    ERROR_CONTACTING_DVA(1004, "Error when contacting DCS for document check"),
+    FAILED_TO_UNWRAP_DVA_RESPONSE(1005, "Failed to unwrap Dcs response"),
+    DVA_RETURNED_AN_ERROR(1006, "DCS returned an error response"),
+    DVA_ERROR_HTTP_30X(1022, "DCS Responded with a HTTP Redirection status code"),
+    DVA_ERROR_HTTP_40X(1023, "DCS Responded with a HTTP Client Error status code"),
+    DVA_ERROR_HTTP_50X(1024, "DCS Responded with a HTTP Server Error status code"),
```

Comment:
Just realised these need updated with new codes

Reply:
and descriptions still some reference to DCS